### ISSUE TITLE
Hands on Training Accessibility Testing Workshop

### ISSUE THREAD

**Issue by teleject** [a11yproject/a11yproject.com#886]
Please provide the following information to streamline adding your event to the site! Delete this boilerplate or any other information that doesn't apply to your event.

Please review our Contributing guidelines:
https://github.com/a11yproject/a11yproject.com/blob/gh-pages/CONTRIBUTING.md#events

## Title
Hands on Training Accessibility Testing Workshop

### Description
Ensuring accessibility into our sites and apps should be seen as a challenge rather than a chore. By working on real-life sites, as well as some code snippets, this workshop will demonstrate an accessibility testing workflow, which can be integrated into your own coding workflow. Also, we will include a review of automated testing, and hands-on manual testing using a variety of tools, from using the keyboard to using a screen reader application.

### URL
https://knowbility.org/services/training/hands-on-training-accessibility-testing-workshop/

### Date
2020-04-02

### Type
Webinar

**Comment by ericwbailey** [comment]
Closed via https://github.com/a11yproject/a11yproject.com/commit/4f670f658b7cba1db58305d7d91e5bf6c4622b2d. Thank you!
